src/interfaces/store-interfaces/state-interface.ts

```diff
@@ -69,6 +69,7 @@ export interface StateModelIF {
   completingParty?: CompletingPartyIF
   parties?: Array<PartyIF>
   restoration: RestorationStateIF
+  isRestorationTypeValid: boolean
```

Comment:
Can this be moved to the restoration state object?

Reply:
I thought about it. Thing is, restoration schema doesn't have an isValid field.

Reply:
There is an interface for the restoration state, which is different from the restoration JSON object.

Reply:
Oh wait, I think I have things mixed up a little. In that case, I think I can move it to the state object.